### PR TITLE
Remove cassandra

### PR DESCRIPTION
**Pull request by manuelgavidia**
This is connected to dojot/dojot#196

### CODE REVIEW

File: LoggingAPI/LoggingAPI-ejb/src/main/java/br/com/dojot/mutualauthentication/loggingapi/dao/impl/ConfigDAOImpl.java [hunk 1]
```diff
@@ -22,11 +22,11 @@ public String findConfigurationByKey(String key) {
 	public List<String> searchCassandraContactPoints() {
```
Comment (raulnegreiros) [inline LoggingAPI/LoggingAPI-ejb/src/main/java/br/com/dojot/mutualauthentication/loggingapi/dao/impl/ConfigDAOImpl.java]: Is this method necessary?

File: LoggingAPI/LoggingAPI-ejb/src/main/java/br/com/dojot/mutualauthentication/loggingapi/dao/impl/ConfigDAOImpl.java [hunk 2]
```diff
@@ -8,9 +8,9 @@
 
 @Singleton
 public class ConfigDAOImpl extends GenericRedisDAOImpl implements ConfigDAO {
-	
+
 	private String CASSANDRA_KEYS = "CASSANDRA_CONTACT_POINTS";
```
Comment (raulnegreiros) [inline LoggingAPI/LoggingAPI-ejb/src/main/java/br/com/dojot/mutualauthentication/loggingapi/dao/impl/ConfigDAOImpl.java]: Is this variable necessary?

File: LoggingAPI/LoggingAPI-ejb/src/main/java/br/com/dojot/mutualauthentication/loggingapi/dao/impl/ConfigDAOImpl.java (hunk 1 above)
Comment (manuelgavidia) [reply LoggingAPI/LoggingAPI-ejb/src/main/java/br/com/dojot/mutualauthentication/loggingapi/dao/impl/ConfigDAOImpl.java]: The whole ConfigDAO interface may not longer be needed, but to purge all mentions to cassandra demands more changes here and in dojot/ma-serviceregistry.
The point of this PR is just to cut off cassandra functionality as a require.

File: LoggingAPI/LoggingAPI-ejb/src/main/java/br/com/dojot/mutualauthentication/loggingapi/dao/impl/ConfigDAOImpl.java (hunk 2 above)
Comment (manuelgavidia) [reply LoggingAPI/LoggingAPI-ejb/src/main/java/br/com/dojot/mutualauthentication/loggingapi/dao/impl/ConfigDAOImpl.java]: I will return a empty list for this methods.

**Pull request merged**